src/style.css

```diff
@@ -1,3 +1,62 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+}
+
+.nav__list {
+  list-style: none;
+  margin: 0;
+  padding-right: 40px;
+  display: flex;
+  flex-direction: row;
+}
+
+.nav__link {
+  color: black;
+  text-decoration: none;
+  padding: 0 10px;
+  font-size: 12px;
+  display: inline-block;
+  line-height: 60px;
+}
+
+.nav__link:hover {
+  cursor: pointer;
+  color: #00acdc;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  text-transform: uppercase;
+  align-items: center;
+  flex-direction: row-reverse;
+}
+
+.moyo-logo {
+  position: absolute;
```

Comment:
Absolute positioning is redundant here. Consider leveraging the standard flow of the document